### DISCUSSION TITLE
Principle: Contributions should generate interest within the member organisation

### DISCUSSION THREAD

**Discussion by ColinEberhardt** [finos/technical-oversight-committee#16] [General]
Why? FINOS wishes to drive both adoption of, and collaboration within their portfolio of projects and standards. A key prerequisite of this is ‘general interest’. Ideally this interest should be expressed at the point of contribution, however, we may accept projects that we feel have the potential to generate interest among the members in the future.

**Comment by jonfreedman** [comment]
Am not sure if this is substantially different from https://github.com/finos/technical-steering-committee/discussions/15 - could also argue that "general interest" is a lesser requirement than "adoption" - e.g. Alice have a general interest in her competitor Bob doing exploratory work in an area so she doesn't have to, but she may not adopt Bob's work even if it is a success as she may have her own internal implementation. I know this isn't really in the spirt of Open Source but it's probably still the reality in a lot of places.

**Reply by ColinEberhardt** [reply]
> Am not sure if this is substantially different from https://github.com/finos/technical-steering-committee/discussions/15

That's a very good point, and I'm inclined to agree

**Reply by ColinEberhardt** [reply]
via our TSC list - "I do see this as different from the first principle – something can be aligned with the purpose but not be of interest. It might be to specific to how an organisation does business or make too many assumptions about process.  It generating interest amongst the TSC seems like a good measure at the moment but there’s a whole host of reasons why it might not get uptake even if it’s a great project – ie it could be something that other already have a proprietary implementation for.  I suspect this is something we will need to see projects through incubation to get a good feel for."

**Comment by rikoe** [comment]
How do we define "generate interest" (if we keep this one)

**Reply by ColinEberhardt** [reply]
Does it generate interest among the TSC? we are a suitable proxy for the rest of the organisation.
